sentinelhub/api/catalog.py

```diff
@@ -106,13 +114,23 @@ def search(
         :param geometry: A search geometry, it will always reprojected to WGS 84 before being sent to the service.
             This parameter is defined with parameter `intersects` at the service.
         :param ids: A list of feature ids as defined in service documentation
-        :param query: A STAC query described in Catalog API documentation
+        :param filter: A STAC filter in CQL2, described in Catalog API documentation
+        :param filter_lang: How to parse CQL2 of the `filter` input, described in Catalog API documentation
```

Comment:
If there is any openly available documentation, I would add a link somewhere in the documentation. Because currently it is not clear for somebody not familiar.

Reply:
I talked with Majcen, he said he didn't find anything useful on CQL2

Reply:
🤷‍♂️  every man and woman for themselves then